interface/main/messages/messages.php

```diff
@@ -23,614 +24,686 @@
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-3-1-1/index.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-11-4/jquery-ui.min.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/qtip2-2-2-1/jquery.qtip.min.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/moment-2-13-0/moment.js"></script>
+    <script>
+      var xljs1 = '<?php echo xl('Preferences updated successfully'); ?>';
+    </script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['web_root']; ?>/interface/main/messages/js/reminder_appts.js"></script>
+    <script type="text/javascript">
+        <?php require_once("$srcdir/restoreSession.php"); ?>
+    </script>
+    <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js">
+    </script>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="MedEx Bank">
+    <meta name="author" content="OpenEMR: MedExBank">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+      label { font-weight:400;}
+      </style>
+  </head>
+
+  <body class="body_top" style="font-size:0.8em;">
+    <div class="row">
+        <?php
+        if (empty($_REQUEST['nomenu'])) $MedEx->display->navigation($logged_in); ?>
+    </div>
+    <?php if (!empty($_GET['go'])) { ?>
+            <?php
+            if (($_REQUEST['go']=="setup")&&(!$logged_in)) {
+                $stage = $_REQUEST['stage'];
+                if (!is_numeric($stage)) {
+                    echo "<br /><br /><span class='title'>$stage ".xlt('Warning: This is not a valid request').".</span>";
+                } else{
+                    $MedEx->setup->MedExBank($stage);
                 }
-                // There's no note ID, and/or it's assigned to the patient.
-                // In these cases a new note is created.
-                addPnote($reply_to, $note, $userauthorized, '1', $form_note_type, $assigned_to, '', $form_message_status);
+            } elseif ($_REQUEST['go']=="addRecall") {
+                $MedEx->display->display_add_recall();
+            } else if ($_REQUEST['go']=='Recalls') {
+                $MedEx->display->display_recalls($logged_in);
+            } elseif ((($_REQUEST['go']=="setup")||($_REQUEST['go']=='Preferences'))&&($logged_in)) {
+                $MedEx->display->preferences();
+            } elseif ($_REQUEST['go']=='icons') {
+                $MedEx->display->icon_template();
+            } elseif ($_REQUEST['go']=='SMS_bot') {
+                $MedEx->display->SMS_bot($logged_in);
+            } else {
+                echo xlt('Warning:  Navigation error.  Please behave.');
```

Comment:
in translation constants, avoid having more than 1 consecutive space (the google docs translation pipeline can sometime get messed up by this)